setup.py

```diff
@@ -6,13 +6,13 @@
 min_py_version = (3, 5)
 
 if sys.version_info <  min_py_version:
-    sys.exit('DataJoint is only supported on Python {}.{} or higher'.format(*min_py_version))
+    sys.exit('DataJoint is only supported for Python {}.{} or higher'.format(*min_py_version))
```

Comment:
this is actually less idiomatic usage.. don't want to go all grammar crazy with a paragraph about this though, both make sense